apps/fabric-website/src/pages/Styles/TypographyPage/docs/web/TypographyCustomization.md

```diff
@@ -0,0 +1,19 @@
+You can add some of your own customizations to the default theme.
+For example, you can utilize `defaultFontStyle` to override the default font style and
+`fonts` to override the default font styles for specific fonts, like small, medium, large, etc.
```

Comment:
```suggestion
`fonts` to override the default font styles for specific font sizes, like small, medium, large, etc.
```

I also just noticed that the [typography page](https://developer.microsoft.com/en-us/fabric#/styles/web/typography) doesn't even mention our defaults for theme sizes like medium, it just has numeric sizes. I'm guessing that may be what the Fluent toolkit is encouraging or something, but it's not very helpful for Fabric developers... @Jahnp Do you think we could add a section on this page for "Theme Fonts" or something like that?

Reply:
@ecraig12345 I understand the suggestion, but I'm not sure if changing this to 'font sizes' is quite accurate, since something like 'medium' or 'large' is actually a font size, font weight, and font family. I'll change this to "font style" maybe?

Reply:
I changed it to "font variants". Let me know what you think!

Reply:
Good idea!